content/id/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Plugin Jaringan
+content_template: templates/concept
+weight: 10
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+{{< warning >}}Fitur-fitur Alpha berubah dengan cepat. {{< /warning >}}
+
+Network plugins in Kubernetes come in a few flavors:
+_Plugin_ jaringan di Kubernet datang dalam beberapa varian:
+
+* _Plugin_ CNI : mengikuti spesifikasi appc / CNI, yang dirancang untuk interoperabilitas.
+* _Plugin_ Kubenet : mengimplementasikan dasar `cbr0` menggunakan _plugin_ `bridge` dan `host-local` CNI
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Instalasi
+
+Kubelet memiliki _plugin_ jaringan bawaan tunggal, dan jaringan bawaan umum untuk seluruh kluster. _Plugin_ ini memeriksa _plugin-plugin_ ketika dijalankan, mengingat apa yang ditemukannya, dan mengeksekusi _plugin_ yang dipilih pada waktu yang tepat dalam siklus pod (ini hanya berlaku untuk Docker, karena rkt mengelola _plugin_ CNI sendiri). Ada dua parameter perintah Kubelet yang perlu diingat saat menggunakan _plugin_:
+
+* `cni-bin-dir`: Kubelet memeriksa direktori ini untuk _plugin-plugin_ saat startup
+* `network-plugin`: _Plugin_ jaringan untuk digunakan dari `cni-bin-dir`. Ini harus cocok dengan nama yang dilaporkan oleh _plugin_ yang diperiksa dari direktori _plugin_. Untuk _plugin_ CNI, ini hanyalah "cni".
```

Comment:
```suggestion
* `network-plugin`: _Plugin_ jaringan untuk digunakan dari `cni-bin-dir`. Ini harus cocok dengan nama yang dilaporkan oleh _plugin_ yang diperiksa dari direktori _plugin_. Untuk _plugin_ CNI, ini (nilainya) hanyalah "cni".
```

We can add this to help the readers refering to the value to pass to the `network-plugin` Kubelet cli parameter

Reply:
done